octobot_trading/enums.py

```diff
@@ -118,8 +123,8 @@ class TransactionType(enum.Enum):
     BLOCKCHAIN_WITHDRAWAL = "blockchain_withdrawal"
     FUNDING_FEE = "funding_fee"
     TRADING_FEE = "trading_fee"
-    REALISED_PNL = "realised_pnl"
-    CLOSE_REALISED_PNL = "close_realised_pnl"
+    REALIZED_PNL = "realized_pnl"
+    CLOSE_REALIZED_PNL = "close_realized_pnl"
```

Comment:
What's the difference between the last position realized pnl value and the close realised pnl ?

Reply:
close realized is the pnl you would have, if you close the current open position
CLOSE_REALIZED_PNL = REALIZED_PNL + UNREALIZED_PNL